.github/workflows/release.yml

```diff
@@ -0,0 +1,56 @@
+name: release to quay.io
+on:
+  push:
+    tags: [v*]
+
+env:
+  REGISTRY_USER: netobserv+github_ci
+  REGISTRY_PASSWORD: ${{ secrets.QUAY_SECRET }}
+  REGISTRY: quay.io/netobserv
+  IMAGE: flowlogs-pipeline
+
+jobs:
+  push-image:
+    name: push image
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        go: ['1.17']
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+      - name: validate tag
+        id: validate_tag
+        run: |
+          tag=`git describe --exact-match --tags 2> /dev/null`
```

Comment:
A question and a suggestion ... we have some of that logic in the version building in the Makfile ... would it make sense to expose that with one a Makefile target instead of writing all this logic here ... maybe move to a script under hack ... I just think that having so much logic inside github action might not be good practice?


This is just a suggestion ... not blocking merge or something like that ,.,., this is good now :-0)

Reply:
Maybe one downside if we have it in a separate place, it's that the `echo "::set-output name=tag::$tag"` command, which is tied to the rest of the github action, would be less visible from the action, so it could be more confusing or error prone if the script is modified. What do you think?